website/docs/r/advanced_cluster.html.markdown

```diff
@@ -600,18 +649,18 @@ If you are upgrading a replica set to a sharded cluster, you cannot increase the
     - Set to `false` to disable disk auto-scaling.
 
 ~> **IMPORTANT:** If `disk_gb_enabled` is true, then Atlas will automatically scale disk size up and down.
```

Comment:
@Zuhairahmed could you please suggest the guidance to include here for using `disk_size_gb` only if using provisioned IOPS as called out by Melissa in the scope? see link in the [ticket description](https://jira.mongodb.org/browse/CLOUDP-260667)

Reply:
Note: Using `disk_size_gb` with Standard IOPS could lead to errors and configuration issues. Therefore, it should be used only with the Provisioned IOPS volume type [https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster#PROVISIONED]. When using Provisioned IOPS, the `disk_size_gb` parameter specifies the storage capacity, but the IOPS are set independently based on the required performance. Ensuring that `disk_size_gb` is used exclusively with Provisioned IOPS will help avoid these issues.

Reply:
hope this helps. also other aspect of Melissa feedback was to removing the `disk_size_gb` in the examples whenever unnecessary. i'm sure you saw this as well, but just highlighting. thanks!

Reply:
yes, no examples include disk_size_gb
Added note regarding provisioned IOPS